doc/source/orbit.rst

```diff
@@ -346,6 +346,52 @@ behavior
 
 .. image:: images/lp-orbit-integration-EzJz.png
 
+.. _fastchar:
+
+Fast orbit characterization
+-----------------------
+
```

Comment:
Somewhere early in this section need to say that this is from Mackereth & Bovy (2018, in prep) which we will later replace with the link. Don't want people to just cite Binney's paper!